content/arduino-cloud/01.getting-started/01.iot-cloud-getting-started/iot-cloud-getting-started.md

```diff
@@ -53,7 +53,7 @@ To use the Arduino IoT Cloud, a **cloud compatible board** is required. You can
 
 The following boards connect to the Arduino IoT Cloud via Wi-Fi.
 
-- [MKR 1000 WiFi](https://store.arduino.cc/arduino-mkr1000-wifi)
+- [MKR 1000 WiFi](/hardware/mkr-1000-wifi)
```

Comment:
The new link is pointing to the docs product page 🤔

Reply:
Why only that one is the affected?

Reply:
Because the product is unavailable in the store, however if there are more links that are not working - we can update them here as well!

Reply:
Oh, I did not check it, now I saw it was redirecting to that page, approving 👍